cmd/kritis/admission/main.go

```diff
@@ -65,8 +56,37 @@ func main() {
 		fmt.Println(version.Commit)
 		return
 	}
+
+	kritisConfigs, err := kritisconfig.KritisConfigs("default")
+	if err != nil {
+		errMsg := fmt.Sprintf("error getting kritis config: %v", err)
+		glog.Errorf(errMsg)
+		return
+	}
+	if len(kritisConfigs) == 0 {
+		glog.Errorf("No KritisConfigs found in any namespace")
+		return
+	} else if len(kritisConfigs) > 1 {
+		glog.Errorf("More than 1 KritisConfig found, will use the 0th object")
+	}
+
+	kritisConf := kritisConfigs[0]
+	// TODO(https://github.com/grafeas/kritis/issues/304): Use CRD validation instead
+	if kritisConf.Spec.MetadataBackend == "" {
+		glog.Errorf("No KritisConfigs MetadataBackend is defined in the spec")
+		return
+	}
+	if kritisConf.Spec.CronInterval == "" {
+		glog.Errorf("No KritisConfigs CronInterval is defined in the spec")
+		return
+	}
+	if kritisConf.Spec.ServerAddr == "" {
+		glog.Errorf("No KritisConfigs ServerAddr is defined in the spec")
+		return
+	}
```

Comment:
Similar to the above comment, here we could assume defaults if nothing is provided?

Reply:
Updated the logic to use the defaults.